INSTALLATION.md

```diff
@@ -25,6 +26,18 @@ in your GitHub settings for Octobox to work.
 * [Downloading subjects](#downloading-subjects)
 
 # Installation
+## Database Selection
+
+Octobox supports a few database adapters. The full list can be found [here](https://github.com/octobox/octobox/blob/master/lib/datbase_config.rb#L2).
```

Comment:
Make this a permalink with a SHA attached to it (Hit Y on GitHub while highlighting that same line)